tests/integration/mininet_tests.py

```diff
@@ -7486,10 +7476,12 @@ def test_lacp_port_down(self):
 
     def test_untagged(self):
         """All untagged hosts in stack topology can reach each other."""
+        self._set_event_log(self.EVENT_LOGGER_TIMEOUT)
         for _ in range(3):
             self.wait_for_all_lacp_up()
             self.verify_stack_hosts()
             self.flap_all_switch_ports()
+        self.wait_until_matching_lines_from_file(r'.+LAG_CHANGE.+', event_log)
```

Comment:
I don't think event_log is defined in this context anymore.  I would address this by creating a new function that's something like:  wait_until_matching_event_log_entry(regex=???)  -- so you never need to expose event_log outside of that other file.

Reply:
Yeah I missed that. Can I return event_log obj from the method instead?

Reply:
because we would be assuming that event_log = os.path.join(self.tmpdir, 'event.log') is the filename _set_event_log is creating. Want to take that assumption out.

Reply:
I am saving the path as a variable name but I think it makes sense to pass it as a argument to the method. event_log is used for different methods in other places and I think we would be writing extra code for a very specific use case.